server/services/rolledOutPrisonService.ts

```diff
@@ -0,0 +1,12 @@
+import { Request, Response } from 'express'
+import ActivitiesService from './activitiesService'
+
+export default function activeRolledPrisons(activitiesService: ActivitiesService) {
+  return (req: Request, res: Response) =>
+    activitiesService.getRolledOutPrisons().then(r => {
+      const activeAgencies = r
+        .filter(item => item.activitiesRolledOut && item.appointmentsRolledOut)
```

Comment:
Just want to confirm the logic here – is it only considered active only if both activities and appointments have been rolled out?

The ticket says:
> For each prison, if either is set to TRUE and we’re past or equal to the date of rollout, return the prison code in a list.

But it's a bit confusing because it also says:
> The single endpoint refers to BOTH Activities and Appointments switches - and assumes that both will continue to be rolled out together.

If we're assuming both will continue to be rolled out together it probably doesn't matter whether it's an AND or OR, but flagging just in case it should be OR.

Reply:
I think the OR is best - if either one, assume its rolled out.

Reply:
Changed to OR Condition.